### DISCUSSION TITLE
Do the BioCro parameters show delayed development in elevated CO2 concentrations?

### DISCUSSION THREAD

**Discussion by justinmcgrath** [biocro/biocro#128] [General]
This is a paper from Joe Castro in Steve's lab a while ago. I had nearly forgot that one of the major findings that growth in elevated CO2 concentration results in more GDD needed to advance through most developmental stages.

BioCro doesn't have the reproductive stages, but do we see something similar for soybean? Does grain filling start later when parameterized for elevated CO2?

https://academic.oup.com/jxb/article/60/10/2945/579088

**Comment by eloch216** [comment]
Short answer: no, ambient CO2 concentration has no influence on soybean phenology in BioCro.

More details:

BioCro's development model for soybean is based on Setiyono et al. (2007) (doi: 10.1016/j.fcr.2006.07.011). This is implemented via the `BioCro:soybean_development_rate_calculator` and `BioCro:development_index` modules. In this model, the rate of soybean development only depends on ambient air temperature and photoperiod length. Flowering occurs when `DVI` reaches 1, and the value of `DVI` is used by the `BioCro:partitioning_coefficient_logistic` module to calculate carbon partitioning coefficients. Generally, the seed will become the largest carbon sink a bit after `DVI` becomes 1, which would probably be a good way to mark the start of "grain filling."

I think that reproducing the effect observed in SoyFACE would require some modifications to the temperature responses in the Setiyono model, many of which are hard-coded in our implementation at the moment.

**Reply by justinmcgrath** [reply]
Yes, it isn't directly modeled, but I was wondering if that fell out of the parameterization. In the soybean BioCro paper, it was parameterized to control CO2 concentrations, and that worked well for predicting growth in elevated CO2 concentrations. Has it also been parameterized for elevated CO2, and the parameters compared? If not, it's not important. I was just curious, and perhaps if it did show differences it might give some insight into how to adjust the partitioning model to respond to CO2 concentration.

**Reply by eloch216** [reply]
Oh, sorry, I misunderstood. I haven't tried parameterizing to elevated CO2 and can't recall hearing anything about it. Maybe @yufenghe22 has tried it?

**Comment by yufenghe22** [comment]
This is interesting! I did not know about the mechanism regarding CO2. The current Soybean-BioCro does not capture that. Re-parameterize partitioning coefficients (they are often being parameterized) does not change the DVI. The DVI is dependent on temperature, day length, maturity_group, and a bunch of constant parameters. Mathematically, we'd need parameters like `Tmin_R0R1, Topt_R0R1, Tmax_R0R1` to vary with CO2 to change the development stages. And yes, all these constant parameters are based on experiments from Setiyono et al. (2007), which was under ambient CO2.

**Reply by eloch216** [reply]
It's been a while since I looked into this in detail, but: I think the value of `DVI` where `kGrain` equals (and then exceeds) `kLeaf` depends on `alphaLeaf` and `betaLeaf`. If we consider this to be the start of "grain filling," then it could occur at a different time when the model is parameterized for elevated CO2, even if the progression of `DVI` is the same. I think maybe this is what Justin means? We wouldn't see a change in flowering time, which is set by the Setiyono model, but we could see a change in the start of grain filling.

**Reply by yufenghe22** [reply]
yes. That "point" might change. The model only defines when DVI >=1, the flowering and grain filling will start. It does not specify which is when. I have not parameterize the model with high CO2, but I will run one test to see how the partitioning coefficients differ.

> think the value of `DVI` where `kGrain` equals (and then exceeds) `kLeaf` depends on `alphaLeaf` and `betaLeaf`. If we consider this to be the start of "grain filling,"

**Reply by justinmcgrath** [reply]
> Re-parameterize partitioning coefficients (they are often being parameterized) does not change the DVI

Is this when allowing things like Tmin_R0R1 to change? I meant that if you parameterize the development parts to the masses in the data set, do the parameters align with what's observed in development? There isn't a direct comparison, but you could check whether the the DVI parameters fit to elevated CO2 feed into the partitioning model such that grain filling occurs later when fit to elevated CO2.

There's also this paper that finds that elevated temperature accelerates vegetative development, but the effect on reproductive development is minimal. Notably, GDD doesn't predict reproductive development. https://academic.oup.com/jxb/article/74/5/1629/6961021. Something like that was the impetus behind the circadian model. 

None of this seems critical, but these things seem notable at least.

**Reply by justinmcgrath** [reply]
I hadn't refreshed this page when writing my last comment and didn't see Ed's one. Yes, I mean something like that.
